terraform/stacks/main/stack.tf

```diff
@@ -116,6 +116,17 @@ module "cf" {
     bucket_prefix = "${var.bucket_prefix}"
 }
 
+module "credhub" {
```

Comment:
Didn't we say we wanted the credhub database to get created in the bosh module? cc @LinuxBozo

Reply:
Since we've made the co-locating decision, yeah, that's where I would make it happen. The only reason to do the module separately is in case we want to eventually run a separate CH for Concourse at some point. Thoughts?

Reply:
I'm all for putting it where it makes sense today and then ripping it out if we decide to SaaS it later. Just unsure about where we put it if not in `modules/credhub` you mean `modules/bosh_vpc`?